test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,76 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class UtilsTest {
+    private static final Object NULL_OBJECT = null;
+
+    @Test
+    public void isAnyNull() {
+        ArrayList<String> emptyList = new ArrayList<String>();
+        ArrayList<String> nonEmptyList = new ArrayList<String>();
+        nonEmptyList.add("A string");
+
+        // empty list
+        assertFalse(Utils.isAnyNull());
+
+        // Any non-empty list
+        assertFalse(Utils.isAnyNull(new Object(), new Object()));
+        assertFalse(Utils.isAnyNull(emptyList));
```

Comment:
If you want to do the "isAnyNull() looks inside containers" check, then we should use the `nonEmptyList` here.